tests/console/postgresql96server.pm

```diff
@@ -34,5 +34,15 @@ sub run {
     destroy_pgsqldb;
 }
 
+sub post_fail_hook {
+    my $self = @_;
+    select_console 'log-console';
+    upload_logs('/tmp/journal_apparmor.log');
+    upload_logs('/var/lib/pgsql/initlog');
+    assert_script_run('tar -capf /tmp/pg_log.tar.xz /var/lib/pgsql/data/pg_log/*');
+    upload_logs('/tmp/pg_log.tar.xz');
```

Comment:
Can these two lines be replaced by `save_and_upload_log`?

Reply:
Not really. `save_and_upload_log` uses `script_run("$cmd | tee $file")` so it wouldn't be the obvious way how to call tar.

Reply:
I tough about `tar -cz /var/lib/pgsql/data/pg_log/*` but this would then print everything, so yeah - better leave it like this.